src/curve.rs

```diff
@@ -257,6 +279,16 @@ impl Identity for CachedPoint {
                      Z:         FieldElement::one(),
                      T2d:       FieldElement::zero() }
     }
+
+    fn is_identity(&self) -> bool {
```

Comment:
same comment as above, but I'm not sure why anyone would ever need to do this on a `CachedPoint`